docs/charts/index.yaml

```diff
@@ -52,8 +52,8 @@ entries:
     created: "2021-06-02T23:05:43.379448+08:00"
     description: A Helm chart for Apollo Portal
     digest: 5586b2597ebe098a26c94273855461891041241628d85dc13e05ce661ff5e7a2
-    home: https://github.com/ctripcorp/apollo
-    icon: https://raw.githubusercontent.com/ctripcorp/apollo/master/apollo-portal/src/main/resources/static/img/logo-simple.png
+    home: https://github.com/apolloconfig/apollo
```

Comment:
There are 4 lines not changed in this file, for example line 23 24 https://github.com/apolloconfig/apollo/blob/c16e96217989825732164a6c1d5edb555e42aeb3/docs/charts/index.yaml#L23-L24

and line 39 40

https://github.com/apolloconfig/apollo/blob/c16e96217989825732164a6c1d5edb555e42aeb3/docs/charts/index.yaml#L39-L40

Please also update those lines.